libvast/src/logical_operator.cpp

```diff
@@ -0,0 +1,31 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2023 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "vast/logical_operator.hpp"
+
+#include "vast/logical_pipeline.hpp"
+
+namespace vast {
+
+auto runtime_logical_operator::detached() const noexcept -> bool {
+  return false;
+}
```

Comment:
This implementation does not match the documentation: `Returns whether any of the logical pipeline's operators are detached.`

Reply:
Oh, and I even commented on it afterwards... Nevermind.